sonic-ycabled/ycable/ycable_utilities/y_cable_helper.py

```diff
@@ -861,24 +861,28 @@ def delete_ports_status_for_y_cable():
         if asic_index is None:
             helper_logger.log_warning(
                 "Got invalid asic index for {}, ignored".format(logical_port_name))
+        
```

Comment:
no need to add this try: catch block, we can remove

Reply:
My thought process was that we need either a try-catch block or an else condition after line 861 to account for the case asic_index == None, because otherwise we'd get a keyError here:

>           if logical_port_name in y_cable_tbl_keys[asic_index]:
E           KeyError: None

Reply:
that will not be caught by try:catch block, we will straight away land into else block and that is being handled